src/WhatToCook.Application/Domain/PlanOfmeals.cs

```diff
@@ -2,20 +2,27 @@
 
 namespace WhatToCook.Application.Domain;
 
+public record RecipePerDay(DateTime Day, Recipe Recipe);
 public class PlanOfMeals
 {
     public string Name { get; set; }
     public int Id { get; private set; }
     public DateTime FromDate { get; private set; }
     public DateTime ToDate { get; private set; }
     public User User { get; set; } = new User() { Email = "mail123@gmail.com" };
-    public IEnumerable<Recipe> Recipes { get; set; }
 
-    public PlanOfMeals(string name, DateTime fromDate, DateTime toDate, IEnumerable<Recipe> recipes)
+    public List<RecipePlanOfMeals> RecipePlanOfMeals { get; set; } = new List<RecipePlanOfMeals> { };
```

Comment:
`public List<RecipePlanOfMeals> RecipePlanOfMeals { get; set; } = new List<RecipePlanOfMeals> { };`

Can be initialized simpler:
`public List<RecipePlanOfMeals> RecipePlanOfMeals { get; set; } = new()};`